### PR TITLE
executor: add region sync state

### PR DESCRIPTION
**Pull request by AndrewDi**
### What problem does this PR solve?
to make user convenient to search region status.

Problem Summary:

### What is changed and how it works?

What's Changed:
retrive region replication state from `pd`

How it Works:
extend information_schema.tikv_region_status table view.

### Related changes

- PR to update `pingcap/docs`/`pingcap/docs-cn`:
- PR to update `pingcap/tidb-ansible`:
- Need to cherry-pick to the release branch

### Check List 

Tests 

- No code

Side effects

- Performance regression
    - Consumes more CPU
    - Consumes more MEM
- Breaking backward compatibility

### CODE REVIEW

**Comment by AndrewDi** [comment]
@winkyao @crazycs520 PTAL

File: infoschema/tables.go [hunk 1]
```diff
@@ -815,6 +815,8 @@ var TableTiKVRegionStatusCols = []columnInfo{
 	{name: "READ_BYTES", tp: mysql.TypeLonglong, size: 21},
 	{name: "APPROXIMATE_SIZE", tp: mysql.TypeLonglong, size: 21},
 	{name: "APPROXIMATE_KEYS", tp: mysql.TypeLonglong, size: 21},
+	{name: "REPLICATIONSTATUS_STATE", tp: mysql.TypeVarchar, size: 16},
```
Comment (wjhuang2016) [inline infoschema/tables.go:818]: Why the size is 16?, I think 64 is more common.

Comment (AndrewDi) [reply infoschema/tables.go:818]: ok

**Pull request merged**